packages/@react-native-windows/cli/src/runWindows/utils/vstools.ts

```diff
@@ -156,25 +156,27 @@ export function addProjectToSolution(
     '\tGlobalSection(SolutionConfigurationPlatforms) = preSolution',
     '\tEndGlobalSection',
     false,
-  ).map(line => line.match(/\s+([\w|]+)\s=/)![1]);
+  ).map(line => line.match(/\s+([\w\s|]+)\s=/)![1]);
 
   const projectConfigLines: string[] = [];
 
   slnConfigs.forEach(slnConfig => {
-    projectConfigLines.push(
-      `\t\t${projectGuid}.${slnConfig}.ActiveCfg = ${
-        project.projectLang === 'cpp'
-          ? slnConfig.replace('x86', 'Win32')
-          : slnConfig
-      }`,
-    );
-    projectConfigLines.push(
-      `\t\t${projectGuid}.${slnConfig}.Build.0 = ${
-        project.projectLang === 'cpp'
-          ? slnConfig.replace('x86', 'Win32')
-          : slnConfig
-      }`,
-    );
+    if (project.projectLang !== 'cpp' || !slnConfig.endsWith('|Any CPU')) {
```

Comment:
Surprised no test failed due to this change, I thought I saw unittest for autolinking updates...

Reply:
@dannyvv  yeah our tests are fairly basic and don't cover these kinds of scenarios (non RNW projects inside a solution)